schemas/2018-04-16/Microsoft.Insights.json

```diff
@@ -7,16 +7,6 @@
     "scheduledQueryRules": {
```

Comment:
It seems that default values for 'scheduledqueryrules_<alertName>_name' and 'components_<ai/workspaceName>_externalid' aren't generated, in @adibiton example while in the previous version it was generated.
How do we fix it?

Reply:
Again I think this is just a difference between live & simulated - there's nothing in the schema file which can influence this.